<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ripes::EditTab</class>
 <widget class="QWidget" name="Ripes::EditTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1185</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QStackedWidget" name="stackedWidget_2">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page_3">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QSplitter" name="splitter">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="childrenCollapsible">
            <bool>false</bool>
           </property>
           <widget class="QWidget" name="verticalLayoutWidget">
            <layout class="QVBoxLayout" name="editorLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Source code</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>Input type:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="setAssemblyInput">
                 <property name="text">
                  <string>Assembly</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="setCInput">
                 <property name="text">
                  <string>C</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="sourceTypeLayout"/>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QStackedWidget" name="editorStackedWidget">
               <property name="currentIndex">
                <number>0</number>
               </property>
               <widget class="QWidget" name="editorPage">
                <layout class="QGridLayout" name="gridLayout_3">
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item row="0" column="0">
                  <widget class="CodeEditor" name="codeEditor"/>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="editorDisabledPage">
                <layout class="QGridLayout" name="gridLayout_5">
                 <item row="0" column="0">
                  <widget class="QFrame" name="gridFrame">
                   <property name="frameShape">
                    <enum>QFrame::Box</enum>
                   </property>
                   <property name="lineWidth">
                    <number>1</number>
                   </property>
                   <layout class="QGridLayout" name="gridLayout_4">
                    <item row="5" column="1">
                     <spacer name="verticalSpacer_2">
                      <property name="orientation">
                       <enum>Qt::Vertical</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>20</width>
                        <height>40</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                    <item row="1" column="2">
                     <spacer name="horizontalSpacer_4">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>40</width>
                        <height>20</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                    <item row="0" column="1">
                     <spacer name="verticalSpacer">
                      <property name="orientation">
                       <enum>Qt::Vertical</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>20</width>
                        <height>40</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                    <item row="1" column="3">
                     <layout class="QGridLayout" name="gridLayout_6"/>
                    </item>
                    <item row="1" column="0">
                     <spacer name="horizontalSpacer_3">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>40</width>
                        <height>20</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                    <item row="1" column="1">
                     <layout class="QGridLayout" name="gridLayout_7">
                      <item row="1" column="1">
                       <widget class="QLabel" name="inputSrcPath">
                        <property name="text">
                         <string>TextLabel</string>
                        </property>
                        <property name="wordWrap">
                         <bool>true</bool>
                        </property>
                       </widget>
                      </item>
                      <item row="1" column="0">
                       <widget class="QLabel" name="label_5">
                        <property name="text">
                         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;File:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                        </property>
                       </widget>
                      </item>
                      <item row="2" column="0">
                       <widget class="QLabel" name="label_7">
                        <property name="text">
                         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Note:&lt;/span&gt; Enabling the editor will clear any currently loaded program from the simulator&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                        </property>
                        <property name="textFormat">
                         <enum>Qt::RichText</enum>
                        </property>
                        <property name="wordWrap">
                         <bool>true</bool>
                        </property>
                       </widget>
                      </item>
                      <item row="0" column="1">
                       <widget class="QLabel" name="curInputSrcLabel">
                        <property name="text">
                         <string>Foo</string>
                        </property>
                       </widget>
                      </item>
                      <item row="0" column="0">
                       <widget class="QLabel" name="label_4">
                        <property name="text">
                         <string>&lt;b&gt;Current input source:&lt;/b&gt;</string>
                        </property>
                        <property name="textFormat">
                         <enum>Qt::RichText</enum>
                        </property>
                       </widget>
                      </item>
                      <item row="2" column="1">
                       <layout class="QHBoxLayout" name="horizontalLayout">
                        <item>
                         <widget class="QPushButton" name="enableEditor">
                          <property name="text">
                           <string>Enable editor</string>
                          </property>
                         </widget>
                        </item>
                        <item>
                         <spacer name="horizontalSpacer_5">
                          <property name="orientation">
                           <enum>Qt::Horizontal</enum>
                          </property>
                          <property name="sizeHint" stdset="0">
                           <size>
                            <width>40</width>
                            <height>20</height>
                           </size>
                          </property>
                         </spacer>
                        </item>
                       </layout>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="verticalLayoutWidget_2">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Executable code</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Expanding</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>View mode:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="binaryViewButton">
                 <property name="text">
                  <string>Binary</string>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">binaryViewButtons</string>
                 </attribute>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="disassembledViewButton">
                 <property name="text">
                  <string>Disassembled</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">binaryViewButtons</string>
                 </attribute>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="ProgramViewer" name="programViewer"/>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_4"/>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CodeEditor</class>
   <extends>QPlainTextEdit</extends>
   <header>editor/codeeditor.h</header>
  </customwidget>
  <customwidget>
   <class>ProgramViewer</class>
   <extends>QPlainTextEdit</extends>
   <header>programviewer.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="binaryViewButtons"/>
 </buttongroups>
</ui>
